Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM ruby:2.7.3
```

Comment:
See if you can get away with using alpine as a base instead. This will avoid you having to install a bunch of sources.list entries since alpine should already have everything you need, and the install size will be significantly smaller as well.